olga/analytics/forms.py

```diff
@@ -0,0 +1,39 @@
+"""
+Forms for received edX installation data.
+"""
+
+from django import forms
+
+from .models import EdxInstallation, InstallationStatistics
+
+# pylint: disable-all
+# flake8: noqa
```

Comment:
Forms are self documenting and easy to implement and code.
Do I need add documentation or disable just documentation errors?